docs/reference/index.md

```diff
@@ -86,3 +87,7 @@ Pebble can send service logs to a Loki server.
 Pebble records events as "notices". In addition to the built-in notices, clients can report custom notices.
 
 * [Notices](notices)
+
+## API
```

Comment:
Maybe this heading can be "Accessing the API", and a sentence about the API (like the other sections have).